data/gui/options_ui.stkgui

```diff
@@ -72,8 +72,12 @@
                 </div>
 
                 <spacer height="4" width="10" />
-
-                <!-- Nothing here yet -->
+                
+                <div layout="horizontal-row" proportion="1" height="fit">
+                    <checkbox id="Split_Screen_Horizontally"/>
+                    <spacer width="20" height="100%" />
+                    <label height="100%" I18N="In the ui settings" text="Split Screen horizontally (At next race)"/>
```

Comment:
To be honest I have no idea how to improve that, but as a regular player, if I just saw that option, I don't think I would understand what it does :/ Not sure if anyone has ideas on how to make this clearer